### ISSUE TITLE
a typo in train_second.py?

### ISSUE THREAD

**Issue by zhouyong64** [yl4579/StyleTTS2#145]
In L409 of train_second.py:
d_loss = dl(wav.detach(), y_rec.detach()).mean()

Is this second "detach()" a typo? Should there be a detach for y_rec here?
